# This message defines a social rule for a robot
string rule  # "normal" / "accelerate" / "decelerate" / "turn"